tests/unit/ContactItem.spec.js

```diff
@@ -0,0 +1,35 @@
+import { mount } from '@vue/test-utils';
+import ContactItem from '../../src/components/Contact/ContactItem';
+import { expect } from 'chai';
+
+describe('ContactItem Component', function() {
+	it('should render anchor if link is provided', function() {
+		//render component
+		const wrapper = mount(ContactItem, {
+			propsData: {
+				link: 'Toto nie je link',
```

Comment:
I don't think we should put Slovak into our codebase - For developing/debugging purposes it's okay, but we definitely should not merge it to master. Please come up with some realistic sample data and fill it here.